generators/add-web-assets/exc-react/templates/src/components/ActionsForm.js

```diff
@@ -0,0 +1,210 @@
+/*<% if (false) { %>
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+<% } %>
+* <license header>
+*/
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import ErrorBoundary from "react-error-boundary";
+import {
+  Flex,
+  Heading,
+  Form,
+  Picker,
+  TextArea,
+  Button,
+  ProgressCircle,
+  Item,
+  Text,
+} from "@adobe/react-spectrum";
+
+import actions from "../config.json";
+import { actionWebInvoke } from "../utils";
+
+const ActionsForm = (props) => {
+  const [state, setState] = useState({
+    actionSelected: null,
+    actionResponse: null,
+    actionResponseError: null,
+    actionHeaders: null,
+    actionHeadersValid: null,
+    actionParams: null,
+    actionParamsValid: null,
+    actionInvokeInProgress: false,
+  });
+
+  console.log(`runtime object: ${props.runtime}`);
```

Comment:
I would move all the logs to the top level file, in app.js ?

Reply:
I think we should keep this log here, as its related to invoking actions
I will move the ErrorBounday to the App.js file

Reply:
sorry I meant the event and not the logs, the events are related to the app as a whole `runtime` refers to the exc runtime

Reply:
Aah, I see it, 
Thanks